m2t-xpand.aggrcon

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="ASCII"?>
 <aggregator:Contribution xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:aggregator="http://www.eclipse.org/cbi/p2repo/2011/aggregator/1.1.0" label="M2T Xpand for 2021-03">
```

Comment:
You might consider disabling the whole contribution.  By leaving the repository enabled (it could also be disabled), you run the risk that some requirement might still resolve to something provided only  by this contributions' p2 repository...

Reply:
is there a guide somewhere how to do that.
i am strugging with the tooling?

Reply:
is the 2nd commit doing that?

Reply:
It is indeed.  Thank you.  🏅